### PR TITLE
✨ Update document types to match templates

### LINKED ISSUE #1016: Add/update document types to match templates
## Feature Justification

We need to update the document types to include the types accepted by the upcoming data validation feature. 

## Proposed Change or Addition

- Update the `fileTypeDetail` obj in `src/common/enum.js` to include the document types implemented in https://github.com/kids-first/kf-api-study-creator/pull/571
- Many of the templated document types (ones with required columns) are much more specific than the general document types. We should consider removing some of the general document types to avoid end user confusion. Perhaps we can keep the `Other` type as a catch-all and the `Sequencing Manifest` type since there is no template for it yet, but will be one in the near future.

### PR DESCRIPTION
**Pull request by znatty22**
Closes #1016 

## Visual Changes

image

### CODE REVIEW

**Comment by znatty22** [comment]
Use different icon for Participant Observations (its the same as the demographics file)

**Review by znatty22 — COMMENTED** [review]
👍 Looks good! Although I can't approve this PR since I opened it. We need an approval from @dankolbman

**Review by dankolbman — APPROVED** [review]
# 🎸

**Pull request merged**
